estimagic/logging/read_log.py

```diff
@@ -35,7 +36,7 @@ def read_optimization_iteration(path, iteration, include_internals=False):
         KeyError: if the iteration is out of bounds.
 
     """
-    database = load_database(path=path, fast_logging=False)
+    database = load_database(**_process_path_or_database(path_or_database))
     optimization_problem = read_last_rows(
```

Comment:
Here we should also use the `read_start_params` function then

Reply:
True. (a675a68)